src/query.ts

```diff
@@ -12,9 +12,7 @@ const edgesFragment = `
 	}
 `;
 
-// interface ArFSTagInterface extends GQLTagInterface {
-// 	name: 'Drive-Id' | 'Folder-Id';
-// }
+type GQLQuery = { query: string };
```

Comment:
We might end using this type on the tests
```suggestion
export type GQLQuery = { query: string };
```